workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetApp/GetAppContextProvider.tsx

```diff
@@ -262,12 +264,27 @@ const GetAppContextProvider: React.FC<GetAppContextProviderProps> = ({
 				value === 'Free'
 		) ?? false;
 
+	const requiresResources = !!product?.productSpecifications.every(
```

Comment:
Some is the correct for this one

```suggestion
	const requiresResources = !!product?.productSpecifications.some(
```